packages/solarflare-theme/package.json

```diff
@@ -0,0 +1,46 @@
+{
+	"name": "solarflare-theme",
+	"displayName": "Solarflare Theme",
+	"version": "0.0.1",
+	"description": "The official VSCode theme extension for Cloudflare.",
+	"categories": [
+		"Themes"
+	],
+	"keywords": [
+		"wrangler",
+		"cloudflare",
+		"workers",
+		"cloudflare workers",
+		"vscode",
+		"theme",
+		"cloudflare theme"
+	],
+	"homepage": "https://github.com/cloudflare/workers-sdk/packages/vscode-theme#readme",
```

Comment:
Should this be solarflare?

Reply:
This also just isn't a valid link 😅 (you probably want a `/tree/main`?)

Reply:
Thanks @Skye-31